app/styles/_components.less

```diff
@@ -101,8 +99,9 @@ pod-donut[mini] {
     // match size setting in podDonut.js
     min-height: 45px;
 		min-width: 45px;
+    // Take mini donut out of flow and position vertically centered in row
     position: absolute !important;
-		top: 2px;
+		transform: translate(0, -24px);
```

Comment:
It looks like these styles are pretty specific to list view. We might want to use mini donut in other contexts.

How does this look on the monitoring page? How does it look at mobile?

Reply:
This is positioning the mini donut within the `<pod-donut>` element. Any positioning of the mini-donut should be set by the container element outside the donut scope.

Random placement tests
<img width="291" alt="screen shot 2018-01-29 at 4 55 28 pm" src="https://user-images.githubusercontent.com/1874151/35536575-4ccf79e8-0515-11e8-859f-4235fb1fcdb2.png">
<img width="294" alt="screen shot 2018-01-29 at 4 53 16 pm" src="https://user-images.githubusercontent.com/1874151/35536576-4cdc4358-0515-11e8-88a6-86beddd78b76.png">
<img width="316" alt="screen shot 2018-01-29 at 4 33 20 pm" src="https://user-images.githubusercontent.com/1874151/35536276-6958f130-0514-11e8-9a69-4d229b73d7db.png">